admin/services/date.service.js

```diff
@@ -4,6 +4,7 @@ const moment = require('moment')
 const gdsFullFormat = 'D MMMM YYYY'
 const gdsShortFormat = 'D MMM YYYY'
 const gdsLongFormat = 'DD MMM YYYY'
+const customFormat = 'dddd DD MMMM'
```

Comment:
If this is mean to be quite English like - then why specify DD rather than just D
Sure Friday 3 November is more natural than Friday 03 November?

Please consider changing this.

Reply:
@jon-shipley changed